problems/strings.js

```diff
@@ -28,7 +42,14 @@ function lastLetter() {}
  * @returns {string} - 'jimmY'
  */
 
-function capitalizeLastLetter() {}
+function capitalizeLastLetter(str) {
+  if(str === "") {
+      return ""
+    } else {
+        newStr = str[str.length - 1].toUpperCase();
```

Comment:
No need for else on this. 
Without declaring newStr it's being made a global variable